### ISSUE TITLE
MultiScale Testing mode

### ISSUE THREAD

**Issue by loveis98** [CSAILVision/semantic-segmentation-pytorch#161]
Hello! Could you explain please, what does "multi-scale" mode mean? I tried to find an answer in the paper, but I didn't.

**Comment by hangzhaomit** [comment]
Multi-scale testing means, resize the input image into multiple scales to feed into the model, and average the scores as final prediction.

**Comment by loveis98** [comment]
Thank you very much! I try to train the model from default arguments of train.py on my own dataset with image size 800*800, with only 2 class, background/foreground (set num_classes=1). And I have no errors, but it trains very very slowly as I suppose, because I even didn't see any statistics with accuracy and so on, although I changed train.py to print stats per 5 iteration. I have no idea, what's wrong.
